aziot/src/internal/check/mod.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::path::PathBuf;
+
+use serde::Serialize;
+use structopt::StructOpt;
+
+mod additional_info;
+mod checks;
+
+pub use additional_info::AdditionalInfo;
```

Comment:
`s/pub/pub(crate)/`

Reply:
Uhh, sure, though it doesn't really matter since there is no top-level `lib.rs`, and all modules are marked private in `main.rs`. But we've had this pub/pub(crate) discussion before, and this is pretty inconsequential, so sure.